locale/sw.yml

```diff
@@ -0,0 +1,1813 @@
+$comment: ""
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE: NODE   |
+# ----------------------
+# << Keywords >>
+__dirname: ""
+__filename: ""
+exports: ""
+module: ""
+require: "" #require()
+
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE: OCAML  |
+# ----------------------
+# << Keywords >>
+and: "" #
+as: "" #
+assert: "" #
+asr: "" #
+begin: "" #
+class: "" #
+constraint: "" #
+do: "" #
+done: "" #
+downto: "" #
+else: "" #
+end: "" #
+exception: "" #
+external: "" #
+false: "" #
+for: "" #
+fun: "" #
+function: "" #
+functor: "" #
+if: "" #
+in: "" #
+include: "" #
+inherit: "" #
+initializer: "" #
+land: "" #
+lazy: "" #
+let: "" #
+lor: "" #
+lsl: "" #
+lsr: "" #
+lxor: "" #
+match: "" #
+method: "" #
+mod: "" #
+module: "" #
+mutable: "" #
+new: "" #
+nonrec: "" #
+object: "" #
+of: "" #
+open: "" #
+or: "" #
+private: "" #
+rec: "" #
+sig: "" #
+struct: "" #
+then: "" #
+to: "" #
+true: "" #
+try: "" #
+type: "" #
+val: "" #
+virtual: "" #
+when: "" #
+while: "" #
+with: "" #
+
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE: PHP    |
+# ----------------------
+# << Keywords >>
+__halt_compiler: ""
+abstract: ""
+and: ""
+array: ""
+as: ""
+break: ""
+callable: ""
+case: ""
+catch: ""
+class: ""
+clone: ""
+const: ""
+continue: ""
+declare: ""
+default: ""
+die: ""
+do: ""
+echo: ""
+else: ""
+elseif: ""
+empty: ""
+enddeclare: ""
+endfor: ""
+endforeach: ""
+endif: ""
+endswitch: ""
+endwhile: ""
+eval: ""
+exit: ""
+extends: ""
+final: ""
+for: ""
+foreach: ""
+function: ""
+global: ""
+goto: ""
+if: ""
+implements: ""
+include: ""
+include_once: ""
+instanceof: ""
+insteadof: ""
+interface: ""
+isset: ""
+list: ""
+namespace: ""
+new: ""
+or: ""
+print: ""
+private: ""
+protected: ""
+public: ""
+require: ""
+require_once: ""
+return: ""
+static: ""
+switch: ""
+throw: ""
+trait: ""
+try: ""
+unset: ""
+use: ""
+var: ""
+while: ""
+xor: ""
+__CLASS__: ""
+__DIR__: ""
+__FILE__: ""
+__FUNCTION__: ""
+__LINE__: ""
+__METHOD__: ""
+__NAMESPACE__: ""
+__TRAIT__: ""
+
+# << Error Messages >>
+
+# ----------------------
+# |  LANGUAGE: PYTHON  |
+# ----------------------
+# << Keywords >>
+  False: "uwongo"
+  None: "hakuna"
+  True: "ukweli"
+  and: "na"
+  as: "kama"
+  assert: "sisitiza"
+  async: "async"
+  await: "ngoja"
+  break: "vunja"
+  class: "darasa"
+  continue: "endelea"
+  def: "fafanua"
+  del: "duta"
+  elif: "vinginevyo"
+  else: "ikiwa-si-ivyo"
+  except: "isipokuwa"
+  finally: "mwishowe"
+  for: "kwa"
+  from: "kutoka"
+  global:  "kimataifa"
+  if: "kama"
+  import: "agiza"
+  in: "ndani"
+  is: "ni"
+  lambda: "lambda"
+  nonlocal: "isiyo-ya-kawaida"
+  not: "si"
+  or: "au"
+  pass: "pita"
+  raise: "inua"
+  return: "rudisha"
+  try: "jaribu"
+  while: "wakati"
+  with: "na"
+  yield: "mavuno"
+
+# << Built-In Functions >>
+
+  abs: "abs"
+  all: "yote"
+  any: "yoyote"
+  ascii: "ascii"
+  bin: "bin"
+  bool: "bool"
+  breakpoint: "mahali-pa-kuvunja"
```

Comment:
I suggest we think of reducing the use of dashed words.

Breakpoint could just be "breakpointi" 😂